rinsar/_process_utilities.py

```diff
@@ -19,7 +19,7 @@
 from rinsar.rsmas_logging import RsmasLogger, loglevel
```

Comment:
Can you remove the `re` import from `_process_utilities.py` as it is no longer needed?